isar/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: sentry_isar
+description: An integration which adds support for performance tracing for the isar package.
+version: 7.13.1
+homepage: https://docs.sentry.io/platforms/flutter/
+repository: https://github.com/getsentry/sentry-dart
+issue_tracker: https://github.com/getsentry/sentry-dart/issues
+
+environment:
+  sdk: '>=2.17.0 <4.0.0'
+  flutter: '>=3.3.0'
+
+dependencies:
+  isar: ^3.1.0+1
+  isar_flutter_libs: ^3.1.0+1 # contains Isar Core
```

Comment:
isn't it enough to have this as a dev dependency?

Reply:
This was how the `Isar` example had the imports. It does work with only `isar_flutter_libs`, so I removed the other like suggested.